.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:7.10
+      
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      # Run install 
+      - run: yarn install
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+        
+      # Run linter
+      - run: yarn lint 
+      # Build 
+      - run: yarn build 
+      # Check `rsync` 
+      - run: which rsync || sudo apt-get install rsync
+      # Add to known hosts
+      - run: mkdir -p ~/.ssh
+      - run: echo $SSH_KNOWN_HOSTS | base64 -d >> ~/.ssh/known_hosts
+      # rsync folder to nginx
+      - run: rsync --delete -avP --rsync-path="sudo -u www-data rsync" ./dist/ <target> 
```

Comment:
Il target è `circleci@developers.italia.it:/apps/www/publiccode-editor.developers.italia.it/web/`